models/attribute-type/src/attribute-enum-type/attribute-enum-type-draft/generator.ts

```diff
@@ -0,0 +1,11 @@
+import { Generator } from '@commercetools-test-data/core';
+import { TAttributeEnumTypeDraft } from '../types';
+
+// https://docs.commercetools.com/api/projects/productTypes#attributeenumtype
+
+export const generator = Generator<TAttributeEnumTypeDraft>({
+  fields: {
+    name: 'enum',
+    values: null,
```

Comment:
This is a required field, will null work here?

Reply:
I'll use an empty array instead 👍